ssl/statem/statem_clnt.c

```diff
@@ -496,8 +490,17 @@ WRITE_TRAN ossl_statem_client_write_transition(SSL *s)
          */
         return WRITE_TRAN_FINISHED;
 
-    case TLS_ST_CR_HELLO_RETRY_REQUEST:
-        st->hand_state = TLS_ST_CW_CLNT_HELLO;
+    case TLS_ST_CR_SRVR_HELLO:
+        /*
+         * We only get here in TLSv1.3. We just received an HRR, so issue a
```

Comment:
Only get here in 1.3 because for <=1.2 the server will write something else after ServerHello before the client gets to write?

Reply:
Yes